source/module_hamilt_lcao/hamilt_lcaodft/FORCE_STRESS.cpp

```diff
@@ -396,28 +396,31 @@ void Force_Stress_LCAO<T>::getForceStress(const bool isforce,
             {
                 GlobalC::ld.save_npy_f(fcs - GlobalC::ld.F_delta, "f_base.npy", GlobalC::ucell.nat); // Ry/Bohr, F_base
 
-                if (GlobalV::GAMMA_ONLY_LOCAL)
+                if(!GlobalV::deepks_equiv) //training with force label not supported by equivariant version now
                 {
-                    const std::vector<std::vector<double>>& dm_gamma
-                        = dynamic_cast<const elecstate::ElecStateLCAO<double>*>(pelec)->get_DM()->get_DMK_vector();
-                    GlobalC::ld.cal_gdmx(dm_gamma[0], GlobalC::ucell, GlobalC::ORB, GlobalC::GridD, isstress);
-                }
-                else
-                {
-                    const std::vector<std::vector<std::complex<double>>>& dm_k
-                        = dynamic_cast<const elecstate::ElecStateLCAO<std::complex<double>>*>(pelec)
-                              ->get_DM()
-                              ->get_DMK_vector();
-                    GlobalC::ld
-                        .cal_gdmx_k(dm_k, GlobalC::ucell, GlobalC::ORB, GlobalC::GridD, kv.nks, kv.kvec_d, isstress);
-                }
-                if (GlobalV::deepks_out_unittest)
-                    GlobalC::ld.check_gdmx(GlobalC::ucell.nat);
-                GlobalC::ld.cal_gvx(GlobalC::ucell.nat);
+                    if (GlobalV::GAMMA_ONLY_LOCAL)
+                    {
+                        const std::vector<std::vector<double>>& dm_gamma
+                            = dynamic_cast<const elecstate::ElecStateLCAO<double>*>(pelec)->get_DM()->get_DMK_vector();
+                        GlobalC::ld.cal_gdmx(dm_gamma[0], GlobalC::ucell, GlobalC::ORB, GlobalC::GridD, isstress);
+                    }
+                    else
+                    {
+                        const std::vector<std::vector<std::complex<double>>>& dm_k
+                            = dynamic_cast<const elecstate::ElecStateLCAO<std::complex<double>>*>(pelec)
+                                ->get_DM()
+                                ->get_DMK_vector();
+                        GlobalC::ld
+                            .cal_gdmx_k(dm_k, GlobalC::ucell, GlobalC::ORB, GlobalC::GridD, kv.nks, kv.kvec_d, isstress);
+                    }
+                    if (GlobalV::deepks_out_unittest)
```

Comment:
I sugest adding {} after if

Reply:
Thanks for the suggestion, but I intend not to modify things that were proved to work well up till now